.travis.yml

```diff
@@ -54,3 +56,5 @@ script:
   - cd docs && make html && cd ..
   # Make sure the library installs.
   - python setup.py install
+    # Make sure benchmarks runs
+  - benchmark_pydy_code_gen.py 2 4
```

Comment:
Since pydy is installed before this runs, it works. I have also tried ``python bin/benchmark_pydy_code_gen.py 2 4``. Okay I will make plot optional.